transaction/core/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use keys::{RistrettoPrivate, RistrettoPublic};
 pub mod account_keys;
 pub mod amount;
 pub mod blake2b_256;
-mod block;
+mod blockchain;
```

Comment:
This PR splits block.rs into a few separate files. I find small files easier to navigate, and easier to get a sense of what has test coverage.